nixos/modules/services/misc/mautrix-telegram.nix

```diff
@@ -0,0 +1,58 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mautrix-telegram;
+
+  configFile = pkgs.runCommand "mautrix-telegram" {
+    buildInputs = [ pkgs.mautrix-telegram pkgs.remarshal ];
+    preferLocalBuild = true;
+  } ''
+    mkdir -p $out
+    remarshal -if json -of yaml \
+      < ${pkgs.writeText "config.json" (builtins.toJSON cfg.configOptions)} \
+      > $out/config.yaml
+
+    ${pkgs.mautrix-telegram}/bin/mautrix-telegram -c $out/config.yaml -g -r $out/registration.yaml
+  '';
+
+in
+{
+  options.services.mautrix-telegram = {
+    enable = mkEnableOption "Mautrix-telegram, a bridge between Matrix and Telegram";
+
+    configOptions = mkOption {
```

Comment:
An example and a default value would be nice here.

In particular, we could set `appservice.database` to `sqlite:////var/lib/mautrix-telegram/mautrix-telegram.db` by default to use an SQLite database placed in the StateDirectory of the service.

Reply:
I can do that but the exemple will be long !